src/systemctl/systemctl.c

```diff
@@ -1458,15 +1459,18 @@ static void output_unit_file_list(const UnitFileList *units, unsigned c) {
                 id_cols = max_id_len;
 
         if (!arg_no_legend && c > 0)
-                printf("%s%-*s %-*s%s\n",
+                printf("%s%-*s %-*s %-*s%s\n",
                        ansi_underline(),
                        id_cols, "UNIT FILE",
                        state_cols, "STATE",
+                       preset_cols, "VENDOR PRESET",
                        ansi_normal());
```

Comment:
hmm, instead of extending the table here manually, any chance i can convince you to rework this to use the table stuff from src/basic/format-table.[ch]? It's the new way how we do this...

(I'll accept a "no, don't have the time", in which case I'll merge this as it is, but I figure i might at least ask? ;-)

(yes, this would be the first use of format-table.[ch] in systemctl)

Reply:
I stumbled upon your TODO item regarding replacing all 'manual' tables with the new `format-table.[ch]` stuff, so I guess I can give it a try (and maybe later convert other tables as well) :-)

Reply:
you know what? let's just merge this, the conversion can be (and probably should be) a separate commit anyway, and it doesn't matter in which order we commit this.

Would love to see the conversion being done though!